### DISCUSSION TITLE
Re-usage of Custom Filters (Calling a js function from the Post Request Filter)

### DISCUSSION THREAD

**Discussion by Artem0ne** [thunderclient/thunder-client-support#1614] [Q&A] [answered]
Hi,

we are going to resolve the **custom filters** and have a question regards it:

![image](https://github.com/user-attachments/assets/1f3d634c-93a8-4556-ad74-0b6257043892)

If I use **Post Request Filter** to call a JavaScript function from the dropdown list, is it considered a Custom Filter? 

Regard,

Artem

**Comment by rangav** [comment, marked as answer]
Yes, It is called a custom filter. This feature is `deprecated` now. We will **remove** this functionality in the future update.

You can import the js file in Tests -> Scripting and call that function.

Import JS files docs
https://docs.thunderclient.com/scripting/import-files

**Comment by Artem0ne** [comment]
Thank you for your response! We’ve already started refactoring our tests.
